app/controllers/resource_comments_controller.rb

```diff
@@ -7,6 +7,12 @@ def create
       user: current_user,
       resource_id:
     )
-    redirect_to(Resource.find(resource_id))
+    redirect_to_resource(resource_id)
+  end
+
+  private
+
+  def redirect_to_resource(resource_id)
```

Comment:
I am against this method.  Why could be worth writing it?

Reply:
It's not.